### PR TITLE
UI-test-for-publicSharing-usingPasswordPolicy

### PR DESCRIPTION
**Pull request by paurakhsharma**
## Description
UI test for public share with password_policy

## Related Issue
#126

## How Has This Been Tested?
Locally

## Types of changes
- [x] Tests

## Checklist:
- [x] Acceptance tests added

### CODE REVIEW

**Comment by paurakhsharma** [comment]
@individual-it, I have made the change, please review it.

File: tests/acceptance/features/webUIEnforcePasswordPolicyOnPublicShare/enforcePasswordPolicyOnPublicShare.feature [hunk 1]
```diff
@@ -0,0 +1,29 @@
+@webUI
+Feature: enforce password_policy on publicly shared link
+
+	As an admin
+	I want to enforce password_policy on publicly shared link
```
Comment (phil-davis) [inline tests/acceptance/features/webUIEnforcePasswordPolicyOnPublicShare/enforcePasswordPolicyOnPublicShare.feature]: a/link/links

File: tests/acceptance/features/webUIEnforcePasswordPolicyOnPublicShare/enforcePasswordPolicyOnPublicShare.feature [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+	So that publicly shared links are protected from un-authorized access
+
+	Background:
+		Given these users have been created:
+			|username|password|displayname|email       |
```
Comment (phil-davis) [inline tests/acceptance/features/webUIEnforcePasswordPolicyOnPublicShare/enforcePasswordPolicyOnPublicShare.feature]: line up the columns

File: tests/acceptance/features/webUIEnforcePasswordPolicyOnPublicShare/enforcePasswordPolicyOnPublicShare.feature [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+			|user1   |ABC123    |User One   |u1@oc.com.np|
+		And the user has browsed to the login page
+		And the user has logged in with username "user1" and password "ABC123" using the webUI
+
+	Scenario: user tries to share a folder with invalid password
+		Given the administrator has enabled the uppercase letters password policy
```
Comment (phil-davis) [inline tests/acceptance/features/webUIEnforcePasswordPolicyOnPublicShare/enforcePasswordPolicyOnPublicShare.feature]: These given steps can go in the background.

File: tests/acceptance/features/webUIEnforcePasswordPolicyOnPublicShare/enforcePasswordPolicyOnPublicShare.feature [hunk 4]
```diff
@@ -0,0 +1,29 @@
+@webUI
+Feature: enforce password_policy on publicly shared link
```
Comment (phil-davis) [inline tests/acceptance/features/webUIEnforcePasswordPolicyOnPublicShare/enforcePasswordPolicyOnPublicShare.feature]: s/link/links

**Pull request merged**